diff --git i/plugin/a.vim w/plugin/a.vim
index 637feb5..dd7ffe1 100644
--- i/plugin/a.vim
+++ w/plugin/a.vim
@@ -557,12 +557,12 @@ comm! -nargs=? -bang IHS call AlternateOpenFileUnderCursor("h<bang>", <f-args>)
 comm! -nargs=? -bang IHV call AlternateOpenFileUnderCursor("v<bang>", <f-args>)
 comm! -nargs=? -bang IHT call AlternateOpenFileUnderCursor("t<bang>", <f-args>)
 comm! -nargs=? -bang IHN call AlternateOpenNextFile("<bang>")
-imap <Leader>ih <ESC>:IHS<CR>
-nmap <Leader>ih :IHS<CR>
-imap <Leader>is <ESC>:IHS<CR>:A<CR>
-nmap <Leader>is :IHS<CR>:A<CR>
-imap <Leader>ihn <ESC>:IHN<CR>
-nmap <Leader>ihn :IHN<CR>
+" imap <Leader>ih <ESC>:IHS<CR>
+" nmap <Leader>ih :IHS<CR>
+" imap <Leader>is <ESC>:IHS<CR>:A<CR>
+" nmap <Leader>is :IHS<CR>:A<CR>
+" imap <Leader>ihn <ESC>:IHN<CR>
+" nmap <Leader>ihn :IHN<CR>
 
 "function! <SID>PrintList(theList) 
 "   let n = 1
