diff --git i/plugin/golang-alternate.vim w/plugin/golang-alternate.vim
index fc09bea..5bc66ca 100644
--- i/plugin/golang-alternate.vim
+++ w/plugin/golang-alternate.vim
@@ -1,11 +1,11 @@
 function! GolangGetAlternateFilename(filepath)
   let fileToOpen = ""
 
-  if empty(matchstr(a:filepath, "_test"))
-    let currentFileRoot = split(a:filepath, ".go$")[0]
+  if empty(matchstr(a:filepath, "_test.go"))
+    let currentFileRoot = a:filepath[0:-4]
     let fileToOpen = currentFileRoot . "_test.go"
   else
-    let currentFileRoot = split(a:filepath, "_test.go$")[0]
+    let currentFileRoot = a:filepath[0:-9]
     let fileToOpen = currentFileRoot . ".go"
   endif
 
